pkg/apis/bus/v1alpha1/events.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v1alpha1
 
 // Event represent the phase of Job, e.g. pod-failed.
+// +kubebuilder:validation:Enum=*;PodFailed;PodEvicted;Unknown;TaskCompleted;OutOfSync;CommandIssued;JobUpdated;TaskFailed
```

Comment:
Please also add `PodPending` and `PodRunning` https://github.com/volcano-sh/apis/commit/b29e1eb72f156328d34572ed84ac42c5d33f18c8#diff-0cef2cb3eec01d5de5a7d03cfa1093366ee92eb98b67d9f5c06eeba26802a329